composer.json

```diff
@@ -38,7 +43,10 @@
     "scripts": {
         "test": [
             "phpunit"
-        ]
+        ],
+        "phpcs": "./vendor/bin/phpcs .",
+        "phpstan": "./vendor/bin/phpstan",
```

Comment:
Would it be useful to have also this script: "phpcs-fix": "./vendor/bin/phpcbf .", like in this PR: [mot-document-module](https://github.com/dvsa/mot-document-module/pull/7/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R35)